SQL/0000-00-00-schema.sql

```diff
@@ -762,6 +802,33 @@ CREATE TABLE `mri_protocol_checks` (
     FOREIGN KEY (`Scan_type`) REFERENCES `mri_scan_type` (`ID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE `mri_protocol_checks_group` (
+    `MriProtocolChecksGroupID`   INT(2) UNSIGNED NOT NULL AUTO_INCREMENT,
+    `Name`                       VARCHAR(255)    NOT NULL,
+    PRIMARY KEY (`MriProtocolChecksGroupID`)
```

Comment:
The primary key is the Name.
Unless duplicate name are required.

Reply:
I'll leave `MriProtocolChecksGroupID` as the PK and use `UNIQUE` for `Name`.

Typically, this table should have <10 entries. Why not leave the ID to be an `INT(2)`?